src/main/java/nextstep/subway/line/domain/Match.java

```diff
@@ -5,11 +5,11 @@
 public enum Match {
     UP, DOWN;
 
-    public static boolean isUP(final Match match) {
-        return Objects.equals(UP, match);
+    public boolean isUP() {
+        return this == UP;
     }
 
-    public static boolean isDOWN(final Match match) {
-        return Objects.equals(DOWN, match);
+    public boolean isDOWN() {
```

Comment:
```suggestion
    public boolean isDown() {
```

카멜케이스로 작성해보면 어떨까요? 😄

Reply:
넵! isDown , isUp 으로 변경하도록 하겠습니다~
감사합니다~